src/libraries/System.Collections.Immutable/src/README.md

```diff
@@ -0,0 +1,10 @@
+## About
+
+This package provides collections that are thread safe and guaranteed to never change their contents, also known as immutable collections. Like strings, any methods that perform modifications will not change the existing instance but instead return a new instance. For efficiency reasons, the implementation uses a sharing mechanism to ensure that newly created instances share as much data as possible with the previous instance while ensuring that operations have a predictable time complexity.
+
+The `System.Collections.Immutable` library is built-in as part of the shared framework in .NET Runtime. The package can be installed when you need to use it in other target frameworks.
```

Comment:
Small nit as I just saw this PR. Customers can and are welcome to use the package on .NETCoreApp as well, i.e. on net7.0. It's already provided inbox but nothing stops you from referencing it, i.e. when you want to reference its patched version from a non patched runtime / SDK.